tests/test_cases/test_array/test_array.py

```diff
@@ -29,7 +29,7 @@ def _check_logic(tlog, hdl, expected):
         tlog.info("   Found {0!r} ({1}) with value=0x{2:X}".format(hdl, hdl._type, int(hdl)))
 
 def _check_str(tlog, hdl, expected):
-    if str(hdl) != expected:
+    if hdl.value != expected:
```

Comment:
Is there a `StrimgObject.__str__` method that we left behind and should remove / replace with `__bytes__`?

Reply:
Checked myself - no, just a very weird overload.